src/main/java/io/personium/client/ServiceCollection.java

```diff
@@ -86,6 +107,60 @@ public void configure(String key, String value, String subject) throws DaoExcept
         rest.setService(this.getPath(), key, value, subject);
     }
 
+    /**
+     * This method configures a set of services.
+     * @param routes list of route
+     * @param subject Value of the service subject
+     * @throws DaoException Exception thrown
+     */
+    public void configure(List<IPersoniumServiceRoute> routes, String subject) throws DaoException {
+        RestAdapter rest = (RestAdapter) RestAdapterFactory.create(this.accessor);
+        StringWriter sw = new StringWriter();
+
+        try {
+            DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
+            factory.setAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, "");
+            factory.setAttribute(XMLConstants.ACCESS_EXTERNAL_SCHEMA, "");
+            DocumentBuilder builder = factory.newDocumentBuilder();
+            DOMImplementation domImpl = builder.getDOMImplementation();
+
+            String nsD = "DAV:";
+            String nsP = "urn:x-personium:xmlns";
+
+            Document document = domImpl.createDocument(nsD, "D:propertyupdate", null);
+            Element elemSet = document.createElementNS(nsD, "D:set");
+            Element elemProp = document.createElementNS(nsD, "D:prop");
+            document.getDocumentElement().appendChild(elemSet);
+            elemSet.appendChild(elemProp);
+
+            Element elemService = document.createElementNS(nsP, "p:service");
+            elemService.setAttribute("language", "JavaScript");
+            elemService.setAttribute("subject", subject);
+            elemProp.appendChild(elemService);
+
+            for (IPersoniumServiceRoute route : routes) {
+                Element elemPath = document.createElementNS(nsP, "p:path");
+                elemPath.setAttribute("name", route.getName());
+                elemPath.setAttribute("src", route.getSrc());
+                elemService.appendChild(elemPath);
+            }
+
+            TransformerFactory tf = TransformerFactory.newInstance();
+            tf.setAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, "");
+            tf.setAttribute(XMLConstants.ACCESS_EXTERNAL_STYLESHEET, "");
+            Transformer transformer = tf.newTransformer();
+            transformer.setOutputProperty(OutputKeys.INDENT, "yes");
+            transformer.setOutputProperty(OutputKeys.METHOD, "xml");
+
+            transformer.transform(new DOMSource(document), new StreamResult(sw));
+
+        } catch (ParserConfigurationException|TransformerException e) {
```

Comment:
```suggestion
        } catch (ParserConfigurationException | TransformerException e) {
```

It's checkstyle warning. I'll update the checkstyle configuration in this project.